addOns/selenium/src/main/java/org/zaproxy/zap/extension/selenium/ExtensionSelenium.java

```diff
@@ -128,6 +132,8 @@ public class ExtensionSelenium extends ExtensionAdaptor {
     private List<WeakReference<ProvidedBrowsersComboBoxModel>> providedBrowserComboBoxModels =
             new ArrayList<>();
 
+    private List<SeleniumScript> javaSeleniumScripts = new ArrayList<>();
```

Comment:
`customSeleniumScripts`? I'd replace "java" with "custom" everywhere. We could also introduce a new interface, that extends the existing one, and give it a better name.